lib/MultiColumnList/RowMeasurer.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { findDOMNode } from 'react-dom';
+import css from './MCLRenderer.css';
+
+class RowMeasurer extends React.Component {
+  static propTypes = {
+    children: PropTypes.node,
+    heightCache: PropTypes.object,
+    measure: PropTypes.bool,
+    onBlur: PropTypes.func,
+    onFocus: PropTypes.func,
+    onMeasure: PropTypes.func,
+    rowIndex: PropTypes.number,
+    staticPosition: PropTypes.bool,
+    style: PropTypes.object,
+  }
+
+  static defaultProps = {
+    onMeasure: () => {}
+  }
+
+  constructor(props) {
+    super(props);
+    this.handleFocus = this.handleFocus.bind(this);
+  }
+
+  componentDidMount() {
+    if (!this.props.staticPosition) this.measureNode();
+  }
+
+  componentDidUpdate() {
+    if (!this.props.staticPosition) this.measureNode();
+  }
+
+  componentWillUnmount() {
+    const {
+      heightCache,
+      rowIndex,
+    } = this.props;
+
+    heightCache.clearRequest(rowIndex);
+  }
+
+  handleFocus() {
```

Comment:
```suggestion
  handleFocus = () => {
```
so it would become possible to get rid of binding in constructor and thus of constructor.